opensrp-opd/src/main/java/org/smartregister/opd/repository/OpdDetailsRepository.java

```diff
@@ -80,6 +79,11 @@ public boolean saveOrUpdate(@NonNull org.smartregister.opd.pojo.OpdDetails opdDe
         return recordId != -1;
     }
 
+    @Override
+    public boolean save(org.smartregister.opd.pojo.OpdDetails opdDetails) {
+        return saveOrUpdate(opdDetails);
```

Comment:
Update this implementation to  contain the save & throw an exception in `saveOrUpdate`

Reply:
i think its the other way around